src/_playground/Routes.js

```diff
@@ -327,10 +343,20 @@ export class Routes extends Component {
             );
         });
 
+        const sideBarClasses = classnames({
+            'frDocs-Sidebar': true,
+            'frDocs-Sidebar--isHidable': !this.state.showSideNav
+        });
```

Comment:
So although this works, the better practice for non-conditional classes is to just include them as comma-separated args outside of the object.  Example:
```js
const sideBarClasses = classnames('frDocs-Sidebar', {
    'frDocs-Sidebar--isHidable': !this.state.showSideNav
});
```
So this syntax would always include `frDocs-Sidebar` and only include `frDocs-Sidebar--isHidable` if the condition was true.

**NOTE:** This would apply to the `docsContentClasses ` block below too.